### PR TITLE
configure streams based on settings

### PR DESCRIPTION
**Pull request by jsilhan**

### CODE REVIEW

File: pydjamodb/connection.py [hunk 1]
```diff
@@ -71,6 +71,8 @@ def create_table(self,
             settings.PYDJAMODB_DATABASE.get('POINT_IN_TIME_RECOVERY', False) if set_point_in_time_recovery is None
             else set_point_in_time_recovery
         )
+        if not stream_specification and settings.PYDJAMODB_DATABASE.get('STREAM_ENABLED'):
+            stream_specification = {'stream_enabled': True, 'stream_view_type': 'NEW_AND_OLD_IMAGES'}
```
Comment (dstlmrk) [inline pydjamodb/connection.py]: We don't need old images. Use `NEW_IMAGE` only.

Comment (jsilhan) [reply pydjamodb/connection.py]: ok, changed

File: pydjamodb/connection.py [hunk 2]
```diff
@@ -71,6 +71,8 @@ def create_table(self,
             settings.PYDJAMODB_DATABASE.get('POINT_IN_TIME_RECOVERY', False) if set_point_in_time_recovery is None
             else set_point_in_time_recovery
         )
+        if not stream_specification and settings.PYDJAMODB_DATABASE.get('STREAM_ENABLED'):
+            stream_specification = {'stream_enabled': True, 'stream_view_type': 'NEW_IMAGE'}
```
Comment (matllubos) [inline pydjamodb/connection.py]: stream_view_type cannot be changed?

Comment (matllubos) [reply pydjamodb/connection.py]: maybe better solution should be set whole stream specification in PYDJAMODB_DATABASE

Comment (matllubos) [reply pydjamodb/connection.py]: PYDJAMODB_DATABASE = {
 ...
  STREAM_SPECIFICATION: {'stream_enabled': True, 'stream_view_type': 'NEW_IMAGE'}
}

Comment (jsilhan) [reply pydjamodb/connection.py]: good idea, changed.

File: pydjamodb/connection.py [hunk 3]
```diff
@@ -71,6 +71,10 @@ def create_table(self,
             settings.PYDJAMODB_DATABASE.get('POINT_IN_TIME_RECOVERY', False) if set_point_in_time_recovery is None
             else set_point_in_time_recovery
         )
+        stream_specification_settings = settings.PYDJAMODB_DATABASE.get('STREAM_SPECIFICATION')
+        if (not stream_specification and stream_specification_settings
+                and stream_specification_settings.get('stream_enabled')):
```
Comment (jsilhan) [inline pydjamodb/connection.py]: if you specify image view type and streams are disabled you get error

Comment (matllubos) [reply pydjamodb/connection.py]: but this is settings problem. Therefore it should be left up to developer..

Comment (jsilhan) [reply pydjamodb/connection.py]: ok, I have changed that but it's little bit more complicated definition in PYDJAMODB_DATABASE settings from main app.

**Pull request merged**